docs/learn-about-neptune/collaborate.rst

```diff
@@ -0,0 +1,299 @@
+
+   Copy the current URL to the clipboard by clicking any **Copy** button that appears. After copying the link, you can paste it, as needed, in an email message, message or other medium.
+
+   **Example**
+
+    .. image:: ../_static/images/core-concepts/metadata_copy.png
+        :target: ../_static/images/core-concepts/metadata_copy.png
+        :alt: Copy URL
+
+- Copying the URL from the address bar.
+
+Details of the URL composition appear in `Link Structure <collaborate.html#link-structure>`_, above.
+
+Experiment details
+^^^^^^^^^^^^^^^^^^
+
+You can share short experiment IDs. An experiment ID is a unique identifier within the project.
+
+In the **Experiment Details** tab, under the **Metadata** section, click the **Copy** button in the ID field.
+
+**Example**
+
+https://ui.neptune.ai/o/USERNAME/org/example-project/e/HELLO-21/details
+
+Experiment charts and other resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can share experiment charts and other resources.
+
+Inside the experiment, there are tabs on the left side: Charts, Logs, Monitoring, Artifacts, Source code, Parameters, Details.
+Each displays specific content that is logged to Neptune for the specific experiment.
+`Each tab has a unique URL <collaborate.html#additions-to-the-project-url>`_.
+
+Projects
+^^^^^^^^
+You can share projects.
+
+**Example**
+
+https://ui.neptune.ai/o/USERNAME/org/example-project/experiments?viewId=d7f80ebe-5bfe-4d12-97c1-2b1e6184a2ed
+
+
+Experiment comparisons
+^^^^^^^^^^^^^^^^^^^^^^
+
+When you compare experiments in the UI, Neptune assigns it a unique URL. Share the URL to show your teammates the exact comparison
+you made.
+
+**Example**
+
+https://ui.neptune.ai/o/USERNAME/org/example-project/compare?shortId=%5B%22HELLO-191%22%2C%22HELLO-197%22%2C%22HELLO-176%22%2C%22HELLO-177%22%2C%22HELLO-123%22%5D&viewId=6013ecbc-416d-4e5c-973e-871e5e9010e9
+
+
+
+    .. image:: ../_static/images/core-concepts/compare_experiments.png
+        :target: ../_static/images/core-concepts/compare_experiments.png
+        :alt: Compare experiments
+
+
+
+Notebook checkpoints
+^^^^^^^^^^^^^^^^^^^^
+
+Every time you make a Notebook checkpoint, Neptune assigns it a unique ID.
+Similarly to other views in Neptune, you can share a link to the particular Notebook checkpoint.
+
+**Example**
+
+https://ui.neptune.ai/o/USERNAME/org/example-project/n/HPO-analysis-with-HiPlot-82bf08ed-c442-4d62-8f41-bc39fcc6c272/d1d4ad24-25f5-4286-974c-c0b08450d5e1
+
+1. Click **Share** in the checkpoint actions menu:
+
+    .. image:: ../_static/images/core-concepts/notebook_checkpoint.png
+        :target: ../_static/images/core-concepts/notebook_checkpoint.png
+        :alt: Notebook checkpoint
+        :width: 400
+
+2. Click **Copy**.
+
+Notebook comparisons
+^^^^^^^^^^^^^^^^^^^^
+You can share a Notebook comparison with your teammates.
+
+The Notebook comparison feature lets you compares two checkpoints site-by-site, like source code. The comparison has a unique link, as well.
```

Comment:
lets you **compare** two checkpoints **side-by-side**

Reply:
Oops. Good catch!